src/app/App.jsx

```diff
@@ -22,6 +23,7 @@ const App = () => (
         <Route path="/subject" component={Subject} exact />
         <Route path="/articles" component={Article} exact />
         <Route path="/resetpassword" component={ResetPassword} exact />
+        <Route path="/newpassword" component={NewPassword} exact />
         <Route path="/create-article" component={CreateArticle} exact />
```

Comment:
@ebenezerdon I noticed I can navigate to this route even without requesting for password change. I think that should not be possible.

Reply:
I think so too. Do you know how I can go about preventing that?

Reply:
You can make the route check if there is a valid resetToken passed to it as a query. If not, it should redirect to some other page. There is a route on our backend for validating resetToken.
There is likely a better way though.

Reply:
Okay, that should work. We'll create a story for it then.